### DISCUSSION TITLE
Audio not switched for some applications

### DISCUSSION THREAD

**Discussion by pizzapim** [LizardByte/community#311] [Sunshine] [answered]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Is your issue described in the documentation?

- [X] I have read the documentation

### Is your issue present in the nightly release?

- [X] This issue is present in the nightly release

### Describe the Bug

I have two profiles for two games, they are identical except for the executable that is launched. However, for one game the audio is correctly muted on the desktop and piped to the Moonlight client, while for the other it is not. The working game is Hogwards Legacy, while the audio for Cities Skylines plays on my desktop's speakers.

### Additional Context

Problem exists for Moonlight-qt client on both Linux and Windows

### Host Operating System

Windows

### Operating System Version

Windows 11 Enterprise 10.0.22621 Build 22621

### Architecture

64 bit

### Sunshine commit or version

0.20.0.17a320bfa5ccf724223c4bf3d157ec27eb33bbf4

### Package

Windows - installer

### GPU Type

AMD

### GPU Model

AMD Radeon RX 6700 XT

### GPU Driver/Mesa Version

31.0.12027.9001

### Config

```Shell
(empty)
```

### Apps

```json
{
    "env": {
        "PATH": "$(PATH);$(ProgramFiles(x86))\\Steam"
    },
    "apps": [
        {
            "name": "Desktop",
            "image-path": "desktop.png"
        },
        {
            "name": "Steam Big Picture",
            "detached": [
                "steam steam:\/\/open\/bigpicture"
            ],
            "image-path": "steam.png"
        },
        {
            "name": "Hogwards Legacy",
            "output": "",
            "cmd": "C:\\Games\\Hogwarts Legacy\\HogwartsLegacy.exe",
            "exclude-global-prep-cmd": "false",
            "elevated": "true",
            "image-path": "hogwards-legacy.png",
            "prep-cmd": [
                {
                    "do": "C:\\Users\\user\\Documents\\MonitorProfileSwitcher_v0700\\MonitorSwitcher.exe -load:C:\\Users\\user\\Documents\\MonitorProfileSwitcher_v0700\\Moonlight.xml",
                    "undo": "C:\\Users\\user\\Documents\\MonitorProfileSwitcher_v0700\\MonitorSwitcher.exe -load:C:\\Users\\user\\Documents\\MonitorProfileSwitcher_v0700\\PC.xml",
                    "elevated": "false"
                }
            ]
        },
        {
            "name": "Cities Skylines",
            "output": "",
            "cmd": "C:\\Program Files (x86)\\Cities Skylines Hotels and Retreats\\Cities.exe",
            "exclude-global-prep-cmd": "false",
            "elevated": "true",
            "prep-cmd": [
                {
                    "do": "C:\\Users\\user\\Documents\\MonitorProfileSwitcher_v0700\\MonitorSwitcher.exe -load:C:\\Users\\user\\Documents\\MonitorProfileSwitcher_v0700\\Moonlight.xml",
                    "undo": "C:\\Users\\user\\Documents\\MonitorProfileSwitcher_v0700\\MonitorSwitcher.exe -load:C:\\Users\\user\\Documents\\MonitorProfileSwitcher_v0700\\PC.xml",
                    "elevated": "false"
                }
            ],
            "image-path": "cities-skylines.png"
        }
    ]
}
```

### Relevant log output

```shell
(I compared log output for both applications, did not see any relevant differences)
```

**Comment by ReenigneArcher** [comment, marked as answer]
A guess, but I think Cities Skylines must be pointing to a specific device.

**Comment by pizzapim** [comment]
@ReenigneArcher I looked through some discussions on Steam and you seem to be right. Cities Skylines apparently always takes the default audio device. Is there a known way to force a particular audio device for an application?

**Reply by ReenigneArcher** [reply]
In Windows Volume Mixer settings
